tests/ttnn/sweep_tests/sweeps/__init__.py

```diff
@@ -199,6 +199,14 @@ def name_to_string(name):
     return table_name
 
 
+def collect_only(*, include):
+    for file_name in sorted(SWEEP_SOURCES_DIR.glob("**/*.py")):
+        sweep_name = get_sweep_name(file_name)
+        if include and sweep_name not in include:
+            continue
+        print(sweep_name)
```

Comment:
Can we refactor this into a generator function and have `run_sweeps` just call this?

Reply:
Sure